profile/arc_cwl_ro_crate.md

```diff
@@ -32,32 +32,31 @@ flowchart TD
         D["Assay=Dataset"] -- "processSequence" --> C
         D -- "hasPart" --> E
 ```
-When adding the workflow run execution context, the "hasPart" field contains all files that are part of the invoced workflow. The "inputs" and "outputs" of the "ComputationalWorkflow" 
-MAY point to the "objects" and "results" of "CreateAction" via "workExample", while the latter point to the former via "exampleOfWork".
+The "inputs" and "outputs" of the "ComputationalWorkflow" MAY point to the "objects" and "results" of "CreateAction" via "workExample", while the latter point to the former via "exampleOfWork".
```

Comment:
I would suggest to put these terms into code snippets instead of quotation marks